src/forms/steps/users.js

```diff
@@ -44,6 +45,15 @@ const users = (intl) => {
               {
                 type: "required",
               },
+              {
+                type: validatorTypes.PATTERN,
+                pattern: "^[a-z_][a-z0-9_-]*$",
+                message:
```

Comment:
Good but needs to be marked for translation.

Reply:
Marked. I've used `intl.formatMessage` because an attempt to use `<FormattedMessage>` lead to this DDF error https://github.com/data-driven-forms/react-forms/issues/1352